app/assets/config/manifest.js

```diff
@@ -0,0 +1,2 @@
+//= link_directory ../javascripts .js
+//= link_directory ../stylesheets .css
```

Comment:
Ah so this is weird. I saw that Kev had[ bumped the publishing components gem](https://github.com/alphagov/govuk_publishing_components/pull/2691) to use sprockets 4, which means this manifest file has been added to the gem. See https://github.com/alphagov/govuk_publishing_components/pull/2691/commits/f20caa5a159917c7fa80827ce102183c56f0f6ed. But feedback is using sprockets 3, and so according to that PR we shouldn't need to do anything here.

I don't think there's any harm adding the manifest file here (but might be worth adding the comment as per the file in the commit above) but cc @kevindew in case it's useful information.

Reply:
As part of this bump sprockets has been upgraded to 4 (see: https://github.com/alphagov/feedback/pull/1396/files#diff-89cade48462044ee1b672dc5f4c3ec250fbd29effcd8932096a23c1283c6731fR374) in this repo, so this will fail without the manifest file.

This file change looks good 👍